src/java/us/kbase/mobu/initializer/test/DockerClientServerTester.java

```diff
@@ -231,369 +231,310 @@ protected static String prepareDockerImage(File moduleDir,
-        lines.add("perl " + new File("test_scripts/perl/test-client.pl").getAbsolutePath() + 
-                " -tests " + configFilePerl.getAbsolutePath() + " -token " + token.getToken() + 
-                " -endpoint " + clientEndpointUrl
-                );
-        TextUtils.writeFileLines(lines, shellFile);
-        {
-            long startTime = System.currentTimeMillis();
-            ProcessHelper ph = ProcessHelper.cmd("bash", shellFile.getCanonicalPath()).exec(
-                    lib2.getCanonicalFile(), null, true, true);
-            System.out.println("(" + (System.currentTimeMillis() - startTime) + " ms)");
-            int exitCode = ph.getExitCode();
-            if (exitCode != 0) {
-                String out = ph.getSavedOutput();  //outSw.toString();
-                if (!out.isEmpty())
-                    System.out.println("Perl client output:\n" + out);
-                String err = ph.getSavedErrors();  //errSw.toString();
-                if (!err.isEmpty())
-                    System.err.println("Perl client errors:\n" + err);
-            }
-            Assert.assertEquals("Perl client exit code should be 0", 0, exitCode);
-        }
-        // JavaScript
-        System.out.print("JavaScript client -> " + serverType + " server ");
-        clInst.install("javascript", async, false, dynamic, "dev", false, 
-                specFile.getCanonicalPath(), "lib2");
-        shellFile = new File(moduleDir, "test_js_client.sh");
-        lines = new ArrayList<String>(Arrays.asList("#!/bin/bash"));
-        lines.addAll(Arrays.asList(
-                "casperjs test " + new File("test_scripts/js/test-client.js").getAbsolutePath() +
-                " --jq=" + new File("test_scripts/js/jquery-1.10.2.min.js")
-                .getAbsolutePath() + " --tests=" + configFile.getAbsolutePath() + 
-                " --endpoint=" + clientEndpointUrl + " --token=\"" + token.getToken() + "\""
-                ));
-        TextUtils.writeFileLines(lines, shellFile);
-        {
-            long startTime = System.currentTimeMillis();
-            ProcessHelper ph = ProcessHelper.cmd("bash", shellFile.getCanonicalPath()).exec(
-                    new File(lib2, moduleName), null, true, true);
-            System.out.println("(" + (System.currentTimeMillis() - startTime) + " ms)");
-            int exitCode = ph.getExitCode();
-            if (exitCode != 0) {
-                String out = ph.getSavedOutput();
-                if (!out.isEmpty())
-                    System.out.println("JavaScript client output:\n" + out);
-                String err = ph.getSavedErrors();
-                if (!err.isEmpty())
-                    System.err.println("JavaScript client errors:\n" + err);
-            }
-            Assert.assertEquals("JavaScript client exit code should be 0", 0, exitCode);
-        }
-    }
+	
+	protected static void testClients(
+			final File moduleDir,
+			final String clientEndpointUrl,
+			final boolean async,
```

Comment:
What's with the shift here? Is this going from spaces -> tabs or something?
Or does git diff just hate my eyes?

Reply:
Can you point out some changes? Because most of this looks identical (except for the JS tests you removed and the CasperJS checks.

Reply:
It's whitespace. Turn off whitespace in the diffs and you should see the changes. It's just removing the casperjs check and the javascript part of the tests

Reply:
Also formatting the func sig

Reply:
IIRC there was a mix of spaces and tabs; some lines had spaces, some tabs, some both

Reply:
> IIRC there was a mix of spaces and tabs; some lines had spaces, some tabs, some both

😱 

And, right, I haven't used that setting in so long. :P